<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Pojedyncze liczby</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Dodaj do listy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Pokaż</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Generowanie masowe</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="spinBox_2">
            <property name="maximum">
             <number>10000000</number>
            </property>
            <property name="singleStep">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Generuj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Pokaż</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Sortowanie</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="6" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Czas[ms]:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="lineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QListWidget" name="listWidget_3"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Lista po sortowaniu</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QListWidget" name="listWidget_4"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Lista przed sortowaniem</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>qSort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_10">
            <property name="text">
             <string>Insertion Sort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>Selection Sort</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Pokaż</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Pokaż2</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Indexing</string>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>Index Sort</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
